test/unit/functionsAsyncAwait.test.js

```diff
@@ -0,0 +1,226 @@
+const {
+  getFilteredsStargazers,
+  getTopFive,
+  sortDateData,
+  getSum,
+  getSumOfReposStars,
+  getReposMoreThanFiveStars,
+  getLastUpdatedRepos,
+} = require("../../scripts/functionsAsyncAwait.js");
+
+const { data } = require("../fixtures/data.js");
+
+describe("Test of functions to get the repos with more than five stars", () => {
+  test("returns an array with all information about the repos with more than 5 stars", () => {
```

Comment:
WDYT?
```suggestion
  test("returns an array with more than 5 stars", () => {
```
Test descriptions must be short and clear
Do this for all the tests